Doc/library/shutil.rst

```diff
@@ -352,6 +352,10 @@ Directory and files operations
       copy the file more efficiently. See
       :ref:`shutil-platform-dependent-efficient-copy-operations` section.
 
+   .. versionchanged:: 3.9
+      Accepts a :term:`path-like object` for both *src* and *dst*.
+
+
```

Comment:
Please follow the convention of the rest of the file and make this a single blank line.